### DISCUSSION TITLE
Project Path should remember where last you were

### DISCUSSION THREAD

**Discussion by CitrusWire** [godotengine/godot-proposals#9827] [Editor]
When creating a new project, it should remember where you last were. Currently it always defaults to user/username/documents, at least on Windows:

![image](https://github.com/godotengine/godot-proposals/assets/72134064/e2365108-7a66-4121-a511-4d9b5f62f45a)

This is a standard file dialog convention - always start where you last were.

**Comment by AThousandShips** [comment]
Thank you for your proposal, consolidating in:
* https://github.com/godotengine/godot-proposals/issues/1312
